BoxContentSDK/BoxContentSDK/Requests/BOXBookmarkShareRequest.h

```diff
@@ -7,6 +7,8 @@
 
 @interface BOXBookmarkShareRequest : BOXItemShareRequest
 
+@property (nonatomic, readonly, strong) NSString *bookmarkID;
```

Comment:
All these strings should really be copied.

Reply:
done